src/olympia/scanners/tests/test_actions.py

```diff
@@ -82,15 +87,25 @@ def test_delay_auto_approval_indefinitely_and_restrict(self):
         _delay_auto_approval_indefinitely_and_restrict(version)
         assert addon.auto_approval_delayed_until == datetime.max
         assert version.needs_human_review
-        assert EmailUserRestriction.objects.filter(email_pattern=user1.email).exists()
-        assert EmailUserRestriction.objects.filter(email_pattern=user2.email).exists()
-        assert EmailUserRestriction.objects.filter(email_pattern=user3.email).exists()
+        assert EmailUserRestriction.objects.filter(
+            email_pattern=user1.email, restriction_type=RESTRICTION_TYPES.SUBMISSION
+        ).exists()
+        assert EmailUserRestriction.objects.filter(
+            email_pattern=user2.email, restriction_type=RESTRICTION_TYPES.SUBMISSION
+        ).exists()
+        assert EmailUserRestriction.objects.filter(
+            email_pattern=user3.email, restriction_type=RESTRICTION_TYPES.SUBMISSION
+        ).exists()
```

Comment:
Should we also be checking that we don't have an `EmailUserRestriction` with a `restriction_type` of `RESTRICTION_TYPES.APPROVAL`, or is that unnecessary for this test? The same goes for `IPNetworkUserRestriction`.

Reply:
Might as well